feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.spec.ts

```diff
@@ -193,6 +247,19 @@ describe('ConfiguratorAttributeDropDownComponent', () => {
     );
   });
 
+  it('should track value prices for the given attribute', () => {
```

Comment:
Please, rephrase the test description, for instance:
`should update value price if there is change`